src/components/utils/spinner-stripes.tsx

```diff
@@ -2,12 +2,13 @@ import React from "react"
 
 interface SpinnerStripesProps {
   dataCy?: string
+  tailwindProps?: string
 }
 
-const SpinnerStripes = ({ dataCy }: SpinnerStripesProps) => {
+const SpinnerStripes = ({ dataCy, tailwindProps = "stroke-white" }: SpinnerStripesProps) => {
```

Comment:
Totally forgot that you can set a default value like that! Gonna quickly do this in my branch 🤫